CONTRIBUTING.md

```diff
@@ -12,9 +12,11 @@ Note: This is the code development repository for *jQuery Core* only. Before ope
 
 ## Getting Involved
 
-We've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/). We're always looking for help identifying bugs, writing and reducing test cases, and documentation.
+[API design principles](https://github.com/jquery/jquery/wiki/API-design-guidelines)
```

Comment:
Maybe we should change the path for it too? `API-design-guidelines` -> `API-design-principles`